END: Add ability to specify custom icon images (#542)
pydata/pydata-sphinx-theme: docs/user_guide/configuring.rst

Configure icon links
====================

You can add icon links to show up to the right of your main navigation bar.

These links take the following form:

.. code:: python

   html_theme_options = {
       ...
       "icon_links": [
           {
               # Label for this link
               "name": "GitHub",
               # URL where the link will redirect
               "url": "https://github.com/<your-org>/<your-repo>",  # required

>                # Icon class (if type=='fontawesome'), or path to local image (if type=='local')

Reviewer: ```suggestion
               # Icon class (if "type": "fontawesome"), or path to local image (if "type": "local")
```
Do not use python syntax because it can be confusing...

               "icon": "fab fa-github-square",
               # Whether icon should be a FontAwesome class, or a local file
               "type": "fontawesome OR local",  # Default is fontawesome
           }
      ]
   }

There are two kinds of icons you can use, described below:

FontAwesome icons
-----------------

`FontAwesome <https://fontawesome.com/>`_ is a collection of icons that are
commonly used in websites. They include both generic shape icons (e.g., "arrow-down"),
as well as brand-specific icons (e.g. "github").

> You can use FontAwesome icons by specifying ``type=="fontawesome"``, and

Reviewer: ```suggestion
You can use FontAwesome icons by specifying ``"type": "fontawesome"``, and
```

               "name": "GitHub",
               "url": "https://github.com/<your-org>/<your-repo>",
               "icon": "fab fa-github-square",
               "type": "fontawesome",
           },
           {
               "name": "GitLab",
               "url": "https://gitlab.com/<your-org>/<your-repo>",
               "icon": "fab fa-gitlab",
               "type": "fontawesome",
           },
           {
               "name": "Twitter",
               "url": "https://twitter.com/<your-handle>",
               "icon": "fab fa-twitter-square",

>                # The default for `type` is `fontawesome` so it is not required here

Reviewer: ```suggestion
               # The default for `type` is `fontawesome` so it is not actually required in any of the above examples as it is shown here
```

   This has already been added for the brands that have *shortcuts*.

Local image icons
-----------------

If you'd like to display an icon image that is not in the FontAwesome icons library,
you may instead specify a path to a local image that will be used for the icon.

> To do so, use ``type=="local"``, and add a path to an image

Reviewer: ```suggestion
To do so, use ``"type": "local"``, and add a path to an image
```
